AndroidSDKPush/src/main/java/com/leanplum/LeanplumPushService.java

```diff
@@ -349,56 +378,34 @@ private static void showNotification(Context context, final Bundle message) {
     if (message.getString("title") != null) {
       title = message.getString("title");
     }
-    final NotificationCompat.Builder notificationCompatBuilder =
-        LeanplumNotificationHelper.getNotificationCompatBuilder(context, message);
+    NotificationCompat.Builder notificationCompatBuilder = null;
+    Notification.Builder notificationBuilder = null;
 
-    if (notificationCompatBuilder == null) {
-      return;
-    }
     final String messageText = message.getString(Keys.PUSH_MESSAGE_TEXT);
-
-    if (defaultIconId == 0) {
-      notificationCompatBuilder.setSmallIcon(context.getApplicationInfo().icon);
-    } else {
-      notificationCompatBuilder.setSmallIcon(defaultIconId);
-    }
-
-    notificationCompatBuilder.setContentTitle(title)
-        .setStyle(new NotificationCompat.BigTextStyle()
-            .bigText(messageText))
-        .setContentText(messageText);
-
     String imageUrl = message.getString(Keys.PUSH_MESSAGE_IMAGE_URL);
-    Notification.Builder notificationBuilder = null;
+    Bitmap bigPicture = null;
     // BigPictureStyle support requires API 16 and higher.
     if (!TextUtils.isEmpty(imageUrl) && Build.VERSION.SDK_INT >= 16) {
-      Bitmap bigPicture = BitmapUtil.getScaledBitmap(context, imageUrl);
-      if (bigPicture != null) {
-        if ((messageText != null && messageText.length() < MAX_ONE_LINE_TEXT_LENGTH) ||
-            customizer != null) {
-          notificationCompatBuilder.setStyle(new NotificationCompat.BigPictureStyle()
-              .bigPicture(bigPicture)
-              .setBigContentTitle(title)
-              .setSummaryText(messageText));
-        } else {
-          notificationBuilder = LeanplumNotificationHelper.getNotificationBuilder(context, message,
-              contentIntent, title, messageText, bigPicture, defaultIconId);
-        }
-      } else {
+      bigPicture = BitmapUtil.getScaledBitmap(context, imageUrl);
+      if (bigPicture == null) {
         Log.w(String.format("Image download failed for push notification with big picture. " +
             "No image will be included with the push notification. Image URL: %s.", imageUrl));
       }
     }
 
-    // Try to put a notification on top of the notification area. This method was deprecated in API
-    // level 26. For API level 26 and above we must use setImportance(int) for each notification
-    // channel, not for each notification message.
-    if (Build.VERSION.SDK_INT >= 16 && !BuildUtil.isNotificationChannelSupported(context)) {
-      //noinspection deprecation
-      notificationCompatBuilder.setPriority(Notification.PRIORITY_MAX);
+    if (Build.VERSION.SDK_INT < 16 || (customizer != null
+        && notificationBuilderCustomizer == null)) {
```

Comment:
are we missing LeanplumPushService. ?

Reply:
no

Reply:
oo it's a private variable. Yeah this will work. I will still recommend using the LeanplumPushService syntax to clarify it's not a local, but quite minor.